doc/man3/EVP_PKEY_size.pod

```diff
@@ -37,11 +41,26 @@ of cryptographic length is specific to the key cryptosystem.
 EVP_PKEY_security_bits() returns the number of security bits of the given
 I<pkey>, bits of security is defined in NIST SP800-57.
 
+EVP_PKEY_get_params() gets arbitrary parameters from the I<pkey>.  The
+available parameters depend on the provider that supports this particular
+I<pkey>.
+See L<OSSL_PROVIDER(3)> for more information on providers.
+See L<OSSL_PARAM(3)> for more information on parameters.
+
+EVP_PKEY_gettable_params() gets a constant B<OSSL_PARAM> array that
+describes the  gettable parameters for the current algorithm implementation,
+i.e. parameters that can be used with EVP_PKEY_get_params().
+See L<OSSL_PARAM(3)> for the use of B<OSSL_PARAM> as parameter descriptor.
+
 =head1 RETURN VALUES
 
 EVP_PKEY_size(), EVP_PKEY_bits() and EVP_PKEY_security_bits() return a
 positive number, or 0 if this size isn't available.
 
+EVP_PKEY_get_params() returns 1 for success or 0 otherwise.
+EVP_PKEY_gettable_params() returns an OSSL_PARAM array on success or NULL on
+error.
+
```

Comment:
This file is missing a `HISTORY` section: if we unify the documentation into this `.pod` file we should add one and document that the new functions are added in `3.0`